### DISCUSSION TITLE
Vercel and Redis

### DISCUSSION THREAD

**Discussion by irt-msummers** [etherCorps/SK-Redis-SessionManager#6] [General]
For your Vercel demo what are you using for a Redis server? Thanks so much!

**Comment by irt-msummers** [comment]
I changed to use Vercel KV and that works locally -- but when I deploy on Vercel, I get 
```
[ioredis] Unhandled error event: Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
    at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:130:17)

```

I am setting 
```
REDIS_URL Development, Preview, Production 
rediss://default:34dcddec3d8f4844....
```

the same that I have in my .env

**Comment by irt-msummers** [comment]
I even tried to hardcode it in session.ts
```
const sessionOptions: ioRedisSessionOptions = {
	redisClient: new Redis('rediss://default:***********kv.vercel-storage.com:34448'),
	//redisClient: new Redis(REDIS_URL),
```

and still getting
```
[ioredis] Unhandled error event: Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)

```

**Reply by theetherGit** [reply]
I can't say anything. i have to review your code so i can see the flow.

**Comment by theetherGit** [comment]
> Redis

I'm using upstash.

**Comment by irt-msummers** [comment]
Thank you -- found it -- I had a Redis() embedded in a +page.server file -- I forgot about when I was trying to ping to see if Redis was alive.  Really appreciate your help!

**Reply by theetherGit** [reply]
happy to help
